### PR TITLE
OIIO, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Needs https://github.com/haikuports/haikuports/pull/6086 to be merged first
Required for OSL

File: media-libs/openimageio/openimageio-2.2.16.0.recipe [hunk 1]
```diff
@@ -91,6 +102,7 @@ BUILD_REQUIRES="
 BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
+	cmd:git
```
Comment (korli) [inline media-libs/openimageio/openimageio-2.2.16.0.recipe:105]: cmd:git shouldn't be needed.

Comment (Begasus) [reply media-libs/openimageio/openimageio-2.2.16.0.recipe:105]: yeah, need to pull it out :)

Comment (Begasus) [reply media-libs/openimageio/openimageio-2.2.16.0.recipe]: Fixed :)

**Pull request merged**